test/LaTeXImportTests.js

```diff
@@ -0,0 +1,297 @@
+            });
+            it('\\left and \\right', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\left[1+1\\right]').toTeX());
+            });
+            it('\\big', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\big[1+1\\big]').toTeX());
+            });
+            it('\\bigg', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\bigg[1+1\\bigg]').toTeX());
+            });
+            it('\\Big', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\Big[1+1\\Big]').toTeX());
+            });
+            it('\\Bigg', () => {
+                assert.equal('4', nerdamer.convertFromLaTeX('2 \\Bigg[1+1\\Bigg]').toTeX());
+            });
+        });
+    });
+
+    context('Spaces', () => {
+        it('thinspace', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\, + \\, 1').toTeX());
+        });
+        it('negative thinspace', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\! + \\! 1').toTeX());
+        });
+        it('medium mu skip', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\> + \\> 1').toTeX());
+        });
+        it('medium mu skip 2', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\: + \\: 1').toTeX());
+        });
+        it('thick mu skip', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\; + \\; 1').toTeX());
+        });
+        it('en space', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\enspace + \\enspace 1').toTeX());
+        });
+        it('quad', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\quad + \\quad 1').toTeX());
+        });
+        it('qquad', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\qquad + \\qquad 1').toTeX());
+        });
+        it('control space', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\ + \\ 1').toTeX());
+        });
+        it('non-breaking space', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 ~ + ~ 1').toTeX());
+        });
+        it('hspace', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\hspace{1cm} + \\hspace{2en} 1').toTeX());
+        });
+        it('hfill', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 + \\hfill 1').toTeX());
+        });
+        it('hphantom', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\hphantom{1cm} + \\hphantom{2en} 1').toTeX());
+        });
+        it('phantom', () => {
+            assert.equal('2', nerdamer.convertFromLaTeX('1 \\phantom{1cm} + \\phantom{2en} 1').toTeX());
+        });
+    });
+})
+
+describe('Alternative Function', () => {
+    context('Triganometry', () => {
+        it('sin', () => {
+            assert.equal('', nerdamer.convertFromLaTeX('\\sin \\pi/2').toTeX());
```

Comment:
The result should be zero
```suggestion
            assert.equal('0', nerdamer.convertFromLaTeX('\\sin \\pi/2').toTeX());
```

Reply:
Actually, that block is redundent and should be removed.